Doc/library/codecs.rst

```diff
@@ -1443,10 +1443,26 @@ to :class:`bytes` mappings. They are not supported by :meth:`bytes.decode`
 | uu_codec             | uu               | Convert the operand using    |                              |
 |                      |                  | uuencode.                    |                              |
 +----------------------+------------------+------------------------------+------------------------------+
+| escape_codec         | escape           | Encode and decode byte       | :func:`codecs.escape_encode` |
+|                      |                  | sequences using escape       | /                            |
+|                      |                  | sequences, similar to        | :func:`codecs.escape_decode` |
+|                      |                  | :func:`repr` of bytes.       |                              |
++----------------------+------------------+------------------------------+------------------------------+
 | zlib_codec           | zip, zlib        | Compress the operand using   | :meth:`zlib.compress` /      |
 |                      |                  | gzip.                        | :meth:`zlib.decompress`      |
 +----------------------+------------------+------------------------------+------------------------------+
 
+.. function:: codecs.escape_encode(input, errors=None)
+
+   Encode *input* using escape sequences. Similar to how :func:`repr` on bytes
+   produces escaped byte values. Returns a tuple of the encoded bytes and
+   the length consumed.
+
+.. function:: codecs.escape_decode(input, errors=None)
+
+   Decode *input* from escape sequences back to the original bytes.
+   Returns a tuple of the decoded bytes and the length consumed.
+
```

Comment:
We should document that `input` needs to be a bytes or buffer compatible object.